spec/end_to_end/without_partner_spec.rb

```diff
@@ -103,8 +21,127 @@
+            { "operation" => "debit", "category" => "rent_or_mortgage", "frequency" => "monthly", "amount" => 100.0 },
+          ],
+        )
+
+        expect(parsed.dig("state_benefits", 0, "name")).to eq "A"
+        expect(parsed.dig("state_benefits", 0, "payments", 0)).to eq({ "date" => "2023-02-15", "amount" => 1.0, "client_id" => "" })
+
+        expect(parsed["vehicles"]).to eq([{
+          "value" => 1.0,
+          "loan_amount_outstanding" => 5.0,
+          "date_of_purchase" => "2021-02-15",
+          "in_regular_use" => false,
+          "subject_matter_of_dispute" => false,
+        }])
+        expect(parsed["capitals"]).to eq({
+          "bank_accounts" => [],
+          "non_liquid_capital" => [{ "value" => 700.0, "description" => "Non Liquid Asset", "subject_matter_of_dispute" => false }],
+        })
+
+        expect(parsed["properties"]).to eq({
+          "main_home" => {
+            "outstanding_mortgage" => 1.0,
+            "percentage_owned" => 1,
+            "shared_with_housing_assoc" => false,
+            "subject_matter_of_dispute" => false,
+            "value" => 1.0,
+          },
+        })
+      }.to_return(
+        body: FactoryBot.build(:api_result, eligible: "eligible").to_json,
+        headers: { "Content-Type" => "application/json" },
+      )
+
+      click_on "Submit"
+
+      expect(stub).to have_been_requested
+    end
+  end
+
+  context "when interacting with the CFE API", :end2end do
+    include_context "with a no-partner, non-passported certificated check"
+
+    it "shows appropriate content on the results screen" do
+      click_on "Submit"
 
-    expect(stub).to have_been_requested
+      ["Your client is likely to qualify for civil legal aid",
+       "We estimate they will have to pay towards the costs of their case:\n£31.07 per month from their disposable income£0.00 lump sum payment from their disposable capital",
+       "Employment income\n£4.33",
+       "Benefits received\nThis does not include Housing Benefit\n£4.33",
+       "Financial help from friends and family\n£866.67",
+       "Student finance\n£8.33",
+       "Total monthly income£883.66",
+       "Employment expenses\nA fixed allowance if your client gets a salary or wage\n£45.00",
+       "Housing costs minus any Housing Benefit payments the household gets\n£100.00",
+       "Dependants allowance\nA fixed allowance deducted for each dependant in the household\n£338.90",
+       "Total monthly outgoings£483.90",
+       "Assessed disposable monthly income\nTotal monthly income minus total monthly outgoings\n£399.76",
+       "Home client lives in\nHome worth\n£1.00Outstanding mortgage\n-£1.001% share of home equity£0.00Assessed value£0.00",
+       "Vehicle 1\nValue£1.00Assessed value£1.00",
+       "Investments and valuables\n£700.00Total capital\n£701.00",
+       "Total assessed disposable capital£701.00"].each do |line|
+        expect(page).to have_content line
+      end
```

Comment:
Is there any value in adding e.g 
`expect(page).not_to have_content("partner...") `
on any of these tests?